CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 <!-- next version -->
 
+## 0.103.0
+
+### 💡 Enhancements 💡
+
+- `collector, target allocator, opamp`: Require Go 1.22 for building the operator (#2757)
+
+### 🧰 Bug fixes 🧰
+
+- `auto-instrumentation`: Fix webserver instrumentation log file name (#2978)
+  
+  Since webserver instrumentation 1.0.4, the configuration log file has been renamed from appdynamics_sdk_log4cxx.xml.template to opentelemetry_sdk_log4cxx.xml.template. The operator upgraded the webserver instrumentation version but haven't change the configuration file name.
+- `target-allocator`: Fixes a bug that didn't automatically create a PDB for a TA with per-node strategy (#2900)
+
+### Components
+
+* [OpenTelemetry Collector - v0.103.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.103.0)
+* [OpenTelemetry Contrib - v0.103.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.103.0)
+* [Prometheus Operator - v0.74.0](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.74.0)
```

Comment:
i was thinking we would add this to our compat matrix in the readme not the components section given this isn't something we release but depend on

Reply:
I thought I'd add it to the release notes so it's easily apparent, but I guess the compat matrix works as well.